airbyte-integrations/connectors/source-mongodb-v2/README.md

```diff
@@ -26,7 +26,29 @@ the Dockerfile.
 We use `JUnit` for Java tests.
 
 ### Test Configuration
-No specific configuration needed for testing, MongoDb Test Container is used.
+
+In order to test the MongoDb source, you need to have a standalone instance, a replica set or Atlas cluster.
```

Comment:
it doesn't work with a docker image anymore?

Reply:
It works, but I've also added test for Atlas Cluster with ssl, I've corrected doc.